src/acb_vector.jl

```diff
@@ -0,0 +1,35 @@
+@inline clear!(acb_vs::acb_vec_struct) = clear!(acb_vs.entries, acb_vs.n)
+Base.size(acb_v::acb_vec_struct) = (acb_v.n,)
+Base.getindex(v::acb_vec_struct, i::Integer) = v.entries + (i - 1) * sizeof(acb_struct)
+function Base.setindex!(v::acb_vec_struct, x, i::Integer)
+    set!(v[i], x)
+    return x
+end
+
+Base.size(v::AcbVector) = size(v.acb_vec)
+Base.precision(v::AcbVector) = v.prec
```

Comment:
I would propose to move this to `precision.jl` and also add `AcbVector` to the other corresponding functions there.